include/dpp/cluster.h

```diff
@@ -2671,7 +2685,7 @@ class DPP_EXPORT cluster {
 	/**
 	 * @brief Edit a role's position in a guild
 	 * 
-	 * Modify the positions of a set of role objects for the guild. Requires the `MANAGE_ROLES` permission.
+	 * Modify the position of a role. Requires the `MANAGE_ROLES` permission.
```

Comment:
instead of doing this the function should be fixed to accept multiple roles like the API call it connects to